queenbee/base/metadata.py

```diff
@@ -26,44 +26,49 @@ class Maintainer(BaseModel):
 
 
 class License(BaseModel):
-    """License information for the Recipe"""
+    """License information for the Package"""
```

Comment:
Is `Package` meant to be more abstract here? So it refers to an operator or recipe?

Reply:
Package is more abstract indeed. A recipe or an operator can both be packaged and the way the package and it's metadata is organised is the same. The only schema difference will be the `manifest` inside the package.